translations/en-US.json

```diff
@@ -51,5 +51,9 @@
   "terraApplication.notifications.newNotificationLabel": "New {label} notification. {bannerType} {bannerDescription}, {bannerAction}, {bannerDismiss}.",
   "terraApplication.notifications.removedNotificationLabel": "Removed {label} notification. {bannerType} {bannerDescription}.",
 
-  "terraApplication.actionMenu.headerCloseButtonLabel": "Close"
+  "terraApplication.actionMenu.headerCloseButtonLabel": "Close",
+
+  "TODO": "translations needed for the below",
+  "terraApplication.skipToLink.prefix": "Skip to {description}"
+
```

Comment:
I would prefer to get translations in one go in a future iteration, but it's gonna be a pain to track what is missing. I have a hack here to try and keep things separated.

The existing translations for the "Skip to Content" string cannot be reused, as some locales have very specific translations that I cannot tokenize. We'll have to see how that shakes out.